src/index.ts

```diff
@@ -1,13 +1,13 @@
-import {firebase, FirebaseDatabaseTypes} from "@react-native-firebase/database";
 import {FormData, FormStatus, FormStatusHandler} from "./types";
+import {getDatabase, Database, push, ref, set, onChildChanged, off, update} from "firebase/database";
+import {FirebaseApp} from "@firebase/app";
 
-let db: FirebaseDatabaseTypes.Module;
-
-export function initClient(databaseName: string, region: string) {
-    db = firebase
-        .app()
-        .database(`https://${databaseName}.${region}.firebasedatabase.app/`);
-
+let db: Database;
+let globalStatusMap: Record<string, string> = {};
+export function initClient(app: FirebaseApp, statusMap?: Record<string, string>) {
```

Comment:
Let's revert this to providing databaseName and region since you can have rtdb on different regions and getDatabase(app) will only return the default rtdb on your app default region

Reply:
Based on firebase documentation, users can set the URL via the firebaseConfig object. https://firebase.google.com/docs/database/web/start#add_the_js_sdk_and_initialize